package.json

```diff
@@ -14,7 +14,6 @@
     "@hideoo/eslint-config": "2.0.1",
     "@hideoo/prettier-config": "2.0.0",
     "@hideoo/tsconfig": "2.0.1",
-    "astro": "3.2.3",
```

Comment:
I do not think that this workspace root dependency was ever used. Maybe a leftover.